docs/TransparentAttr.rst

```diff
@@ -14,13 +14,13 @@ This has several consequences:
   doing dataflow-related diagnostics, even under ``-Onone``. This may be
   necessary to *catch* dataflow errors.
 
-- Because of this, a ``@_transparent`` function is inherently "fragile", in
+- Because of this, a ``@_transparent`` function is implicitly inlinable, in
   that changing its implementation most likely will not affect callers in
   existing compiled binaries.
 
 - Because of this, a ``@_transparent`` function MUST only reference public
   symbols, and MUST not be optimized based on knowledge of the module it's in.
-  [This is not currently implemented or enforced.]
+  [The former is caught by checks in Sema.]
```

Comment:
Both of these are only true for public transparent functions. Internal and private transparent can reference private symbols.

Reply:
Ah, right, thanks.